ui/src/config.js

```diff
@@ -16,6 +16,10 @@ if (!isProduction) {
 }
 assetPath += '/';
 
+/** @returns {string} */
+export const getWsUrl = () =>
+  defaultTo(process.env.WS_URL, process.env.SITE_URL.replace(/^http/, 'ws'));
```

Comment:
@ryansmith94 We are assuming a prefix of `http` here

What happens with a `SITE_URL` of `https://saas.learninglocker.net`, for example

Reply:
That's not true if it were "https" it would be replaced with "wss" which according to Chris works fine.

Reply:
`ws://` and `wss://` are like `http://` and `https://`

I.e. `ws://` are insecure and `wss://` are secure

The current code actually works out very nicely then, since it will use `ws://` in development and `wss://` in production 👍